cmd/kubelet/app/server.go

```diff
@@ -129,6 +130,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(&s.ClusterDNS, "cluster_dns", "IP address for a cluster DNS server.  If set, kubelet will configure all containers to use this for DNS resolution in addition to the host's DNS servers")
 	fs.BoolVar(&s.ReallyCrashForTesting, "really_crash_for_testing", s.ReallyCrashForTesting, "If true, crash with panics more often.")
 	fs.DurationVar(&s.StreamingConnectionIdleTimeout, "streaming_connection_idle_timeout", 0, "Maximum time a streaming connection can be idle before the connection is automatically closed.  Example: '5m'")
+	fs.StringVar(&s.ContainerRuntimeChoice, "container_runtime_choice", "docker", "The choice fo the container runtime. Example: 'docker', 'rocket'.")
```

Comment:
"choice" seems superfluous, how about just ContainerRuntime/container_runtime
also s/fo/of/